Essentials/src/main/java/com/earth2me/essentials/Essentials.java

```diff
@@ -903,7 +903,7 @@ public boolean onCommandEssentials(final CommandSender cSender, final Command co
             } catch (final NotEnoughArgumentsException ex) {
                 if (getSettings().isVerboseCommandUsages() && !cmd.getUsageStrings().isEmpty()) {
                     sender.sendTl("commandHelpLine1", commandLabel);
-                    sender.sendTl("commandHelpLine2", command.getDescription());
+                    sender.sendTl("commandHelpLine2", sender.tl(command.getName() + "CommandDescription"));
```

Comment:
this will throw a `MissingResourceException` if the user's locale doesn't have a translation for the given description string, maybe do something like this


```suggestion
                    String description = command.getDescription();
                    try {
                        description = sender.tl(command.getName() + "CommandDescription");
                    } catch (MissingResourceException ignored) {
                    }
                    sender.sendTl("commandHelpLine2", description);
```